src/components/ui/SearchBox/index.jsx

```diff
@@ -0,0 +1,13 @@
+import TextInput from "@components/ui/form/TextInput";
+
+function SearchBox() {
+    const PLACEHOLDER = "Search...";
```

Comment:
Are we using any separate file for managing all constants? Is so then move it accordingly. Otherwise it is good practice to store all constant in one separate helper file.

Reply:
But here, this placeholder text will always be used only for this specific component. So, this is just a local constant variable. After reviewing this code, I have come up with the idea that I should make this `placeholder` a `prop` for this `SearchBox` component and give it a default value of “search...“.
Kindly review it and suggest the best practice here.
Thank you.